utam-core/src/main/java/utam/core/framework/context/MobilePlatformType.java

```diff
@@ -86,4 +86,25 @@ public String getConfigName(String moduleName) {
     return getProfileConfigName(this, moduleName);
   }
 
+  private static boolean isiPad(WebDriver driver) {
+    String device =  ((AppiumDriver) driver).getSessionDetail("device").toString();
```

Comment:
this method can be one line - `return "iPad".equalsIgnoreCase(((AppiumDriver) driver).getSessionDetail("device"))`

Reply:
updated